content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -97,87 +97,160 @@ still considered alpha and may change in future versions.
 
 It's possible to configure `kubeadm init` with a configuration file instead of command
 line flags, and some more advanced features may only be available as
-configuration file options.  This file is passed in the `--config` option.
+configuration file options. This file is passed in the `--config` option.
+
+In Kubernetes 1.11 and later, the default configuration can be printed out using the
+[kubeadm config print-default](/docs/reference/setup-tools/kubeadm/kubeadm-config/) command.
+
+For more details on each field in the configuration you can navigate to our
+[API reference pages.] (https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#MasterConfiguration)
+
+Example of the kubeadm MasterConfiguration version `v1alpha2`:
 
 ```yaml
-apiVersion: kubeadm.k8s.io/v1alpha1
-kind: MasterConfiguration
 api:
-  advertiseAddress: <address|string>
-  controlPlaneEndpoint: <string>
-  bindPort: <int>
+  advertiseAddress: 192.168.0.102
+  bindPort: 6443
+  controlPlaneEndpoint: ""
+apiVersion: kubeadm.k8s.io/v1alpha2
```

Comment:
Normally we put `apiVersion` and `kind` in the beginning two lines.

Reply:
will adjust later today. thanks @dixudx .